packages/api/src/external/commonwell/admin/recreate-patients-at-hies.ts

```diff
@@ -143,7 +143,7 @@ export async function recreatePatientAtCW(
   } catch (error) {
     const msg = `Error while recreating patient at CW: ${error}`;
     log(msg);
-    capture.error(error, { extra: { patient, error } });
+    capture.message(msg, { extra: { patient, error }, level: "error" });
```

Comment:
It's important that the messages are not dynamic when sending to Sentry, so diff events can be associated w/ the same issue.

In this case, I'm assuming that we want all errors associated with the text `Error while recreating patient at CW`, right?

Reply:
Also, `patient` can be quite large, right? I'd consider sending only the ID and what else is relevant. Sentry has a limit on the payload, and `error` is already quite large, so this would minimize the risk of us losing the notification.

Reply:
Yes. I was aware of the dynamic titles - you mentioned it a couple of times to me already. Just overlooked this one since I changed this in like 40 places 😅